### PR TITLE
Completed dom-querying exercise  -  Omid Asadi 

### PR DESCRIPTION
**Pull request by omidzed**
image

### CODE REVIEW

File: dom-querying/main.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+const $explanation = document.querySelector('#explanation');
+console.log('$explanation: ', $explanation);
+
+console.dir($explanation);
+
+const $hint = document.querySelector('.hint');
+console.log('$hint: ', $hint);
+console.dir($hint);
+
+const $paragraphs = document.querySelectorAll('.p');
```
Comment (shawnkost) [inline dom-querying/main.js:17]: This should just be `p` without the `.`. `.p` means you're querying for a class p rather than the `p` element

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here Omid!

File: dom-querying/main.js (hunk 1 above)
Comment (omidzed) [reply dom-querying/main.js]: Made the  change,  and here is the updated screenshot showing the NodeList

image

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**